.nf-core.yml

```diff
@@ -1,2 +1,18 @@
+bump_version: null
+lint: null
+nf_core_version: 3.0.2
+org_path: null
 repository_type: pipeline
-nf_core_version: "2.14.1"
+template:
+  author: James A. Fellows Yates, Sofia Stamouli, Moritz E. Beber, and the nf-core/taxprofiler
```

Comment:
You definitely need to add yourself here @LilyAnderssonLee ! and I think also needs to go in the `nextflow.config` and license!